cmd/botkube/main.go

```diff
@@ -104,6 +104,11 @@ func run(ctx context.Context) error {
 		logger.Warnf("Configuration validation warnings: %v", confDetails.ValidateWarnings.Error())
 	}
 
+	restCfg, err := rest.InClusterConfig()
```

Comment:
This will fail when not run in a kubernetes pod. Add a flag/switch, eg rbac defined?

Reply:
I did same thing here https://github.com/kubeshop/botkube/blob/3e1d8a99c8a18244d2203c12a42509e9b7799280/internal/source/kubernetes/client.go#L28